src/test/java/org/xstefank/check/AuxiliaryTestMethods.java

```diff
@@ -0,0 +1,25 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.xstefank.model.yaml.Format;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.io.File;
+import java.io.IOException;
+
+public class AuxiliaryTestMethods {
+
+    public static JsonNode readJsonNode(String resource) throws IOException {
+        File testFile1 = new File(SkipCheck.class.getClassLoader().getResource(resource).getFile());
+        return new ObjectMapper().readTree(testFile1);
+    }
+
+    public static FormatConfig helpSetUpSkipPatterns(SkipPatterns testSkipPatterns) {
```

Comment:
this method returns configured format configuration with skipPatterns if I understand correctly. It doesn't help with anything :)